cirq/testing/equivalent_repr_eval_test.py

```diff
@@ -19,7 +19,7 @@
 
 
 def test_external():
-    for t in [1, 2, 3, 'a', 0.5, 2**0.5, 1.1, 1j]:
+    for t in ['a', 1j]:
```

Comment:
Dropped floats and ints because, well, they don't satisfy the property that we are holding ourselves to.